packetbeat/protos/redis/redis_parse.go

```diff
@@ -423,7 +424,7 @@ func (p *parser) parseArray(depth int, buf *streambuf.Buffer) (common.NetString,
 	}
 
 	// handle top-level request command
-	if depth == 0 && isRedisCommand(content[0]) {
+	if depth == 0 && (isRedisCommand(content[0]) || (count > 1 && isRedisCommand(bytes.Join(content[0:2], []byte(" "))))) {
 		p.message.isRequest = true
 		p.message.method = content[0]
 		if len(content) > 1 {
```

Comment:
It seems like `p.message.method` and `p.message.path` need to be computed differently as well to handle the two-word case

Reply:
I will have a look and test again.